Classes/Events/Event.py

```diff
@@ -807,21 +808,11 @@ async def add_position(self, interaction: Interaction) -> None:
 
         log.info(self.guild, f"Adding position to event {self.name} ({self.id}).")
 
-        raw_options = self._mgr.guild.position_manager.select_options()
-        options = [
-            o
-            for o in raw_options
-            if o.value not in [p.position.id for p in self.positions]
-        ]
-        if not options:
-            log.debug(self.guild, "No positions available to add.")
```

Comment:
It looks like this logging message is missing from the EventPositionSelectView.py equivalent.

Reply:
Since the SelectMenu in `EventPositionSelectView.py` has a couple lines that handle an empty option list, I just took out the internal log message.